amundsen_application/static/js/components/PreferencesPage/PreferenceGroup/index.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+export type PreferenceGroupProps = {
+  changePreference: any;
+  selected: boolean;
+  title: string;
+  subtitle: string;
+}
+
+export class PreferenceGroup extends React.Component<PreferenceGroupProps> {
+    public static defaultProps: Partial<PreferenceGroupProps> = {
+      selected: false,
+      title: '',
+      subtitle: '',
+    };
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <label className="preference-group" onClick={() => this.props.changePreference(this.props.title)}>
```

Comment:
This is a bit uncertain to review because I recognize we're sort of just making some easily changed assumptions about the way the Redux support will be set up, but at minimum I think we should be working with some sort of `id`. 

The reason is that title can be changed any time, but the Redux layer and the backend support will need some way of knowing what option this is regardless of what the UI is displaying. So we need some `id` value, and then we can more appropriately call `this.props.changePreference(this.props.id)`